en/asgardeo/docs/apis/restapis/actions.yaml

```diff
@@ -0,0 +1,859 @@
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      x-codeSamples:
+        - lang: Curl
+          source: |
+            curl --location --request POST 'https://api.asgardeo.io/t/{organization-name}/api/server/v1/actions/{actionType}/{actionId}/activate' \
+            -H 'Authorization: Bearer {bearer_token}'
+
+  /actions/{actionType}/{actionId}/deactivate:
+    post:
+      tags:
+        - Actions Endpoint
+      summary: Deactivate Action
+      description: "This API deactivates an action using the action's type and unique ID. \n\n
+      - <b>Scope(Permission) required:</b> ``internal_action_mgt_update``\n\n"  
+      operationId: deactivateAction
+      parameters:
+        - name: actionType
+          in: path
+          description: Name of the Action Type.
+          required: true
+          schema:
+            enum:
+              - preIssueAccessToken
+              - preUpdatePassword
+              - preUpdateProfile
+              - preRegistration
+        - name: actionId
+          in: path
+          description: Id of the Action.
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Action is deactivated
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ActionBasicResponseInactive'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      x-codeSamples:
+        - lang: Curl
+          source: |
+            curl --location --request POST 'https://api.asgardeo.io/t/{organization-name}/api/server/v1/actions/{actionType}/{actionId}/deactivate' \
+            -H 'Authorization: Bearer {bearer_token}'
+
+  /actions/{actionType}/{actionId}/{authType}:
+    put:
+      tags:
+        - Actions Endpoint
+      summary: Update Action Authentication
+      description: "This API updates action endpoint authentication and return the updated action configurations. \n\n
```

Comment:
Updates authentication mechanism for the endpoint configured for the action

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4698/commits/ee4f5ca7c24d3d8dcaca4943b7b5aadb6e16e7fc